monitoring/tools/s2_gap_filler.py

```diff
@@ -146,17 +146,15 @@ def publish_message(files: list, queue_name: str, slack_url: str = None) -> str:
 @click.command("s2-gap-filler")
 @click.argument("idx", type=int, nargs=1, required=True)
```

Comment:
You're better off adding docs to this, @carioca-au

Reply:
Arguments do not receive **help** as per docs the Python docs block will be presented as part of the help
https://click.palletsprojects.com/en/8.0.x/documentation/#documenting-arguments